src/gql/queries/project.ts

```diff
@@ -32,6 +32,26 @@ export const GET_TEAM_PROJECTS = gql`
 export const GET_PROJECTS = gql`
   query GetProjects($teamId: ID!, $first: Int, $last: Int, $after: Cursor, $before: Cursor) {
     projects(teamId: $teamId, first: $first, last: $last, after: $after, before: $before) {
+      edges {
+        node {
+          id
+          name
+          description
+          imageUrl
+          isArchived
+          isBasicAuthActive
+          basicAuthUsername
+          basicAuthPassword
+          publicTitle
+          publicDescription
+          publicImage
+          alias
+          publishmentStatus
+          scene {
+            id
+          }
+        }
+      }
```

Comment:
Same as in queries/dataset.ts, since we add the `edges{ node {....` do we still need the `nodes {...` below? They seem to give the same info, so if we don't utilize nodes, only edges { nodes ..., please clean up the queries by removing what we don't need to retrieve